addons/l10n_cl/models/account_move.py

```diff
@@ -58,6 +58,10 @@ def _check_document_types_post(self):
             vat = rec.partner_id.vat
             country_id = rec.partner_id.country_id
             latam_document_type_code = rec.l10n_latam_document_type_id.code
+            if (rec.journal_id.type == 'purchase' and tax_payer_type == '4' and country_id.code != 'CL' and
+                rec.l10n_latam_document_type_id.code == '61' and
```

Comment:
```suggestion
                latam_document_type_code == '61' and
```
We define the variable just above it.